posix.mak

```diff
@@ -31,8 +31,11 @@
 QUIET:=
 
 DEBUGGER=gdb
+GIT_HOME=https://github.com/dlang
+DMD_DIR=../dmd
 
-include osmodel.mak
+$(shell [ ! -d $(DMD_DIR) ] && git clone --depth=1 $(GIT_HOME)/dmd $(DMD_DIR))
```

Comment:
@CyberShadow this was needed to pass on DAutoTest. I assumed that you clone all repos on start?
Anyhow I doubt that this hurts anyone and at best it helps a few newcomers.